.travis.yml

```diff
@@ -2,9 +2,8 @@ language: smalltalk
 sudo: false
 
 smalltalk:
-  - Squeak32-trunk
+  - Squeak-trunk

```

Comment:
Argh, these changes haunt us by now :D The original configuration is correct and should be preserved.